src/DocumentDB.ChangeFeedProcessor/FeedProcessing/PartitionProcessor.cs

```diff
@@ -94,6 +95,9 @@ public async Task RunAsync(CancellationToken cancellationToken)
                             this.options.MaxItemCount /= 2;
                             this.logger.WarnFormat("Reducing maxItemCount, new value: {0}.", this.options.MaxItemCount);
                             break;
+                        default:
```

Comment:
The message should say about unrecognized enum value, not the exception.

Reply:
Also can we add Debug.Fail(same message)?

Reply:
Done